<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>purchaseDialog</class>
 <widget class="QDialog" name="purchaseDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>691</width>
     <height>521</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/images/images/middle.jpg</pixmap>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>691</width>
     <height>521</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tabWidgetPage1">
    <attribute name="title">
     <string>进货信息查询</string>
    </attribute>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>651</width>
       <height>241</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>471</width>
       <height>30</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QComboBox" name="search_type">
        <item>
         <property name="text">
          <string>商品条码</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>供货商</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>进货日期</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="s_id">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="search">
        <property name="text">
         <string>查询</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QToolButton" name="toolButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>370</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string> 导出到 Excel</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabWidgetPage2">
    <attribute name="title">
     <string>进货方案制定</string>
    </attribute>
    <widget class="QPushButton" name="in_submit">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>310</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>确认进货</string>
     </property>
    </widget>
    <widget class="QPushButton" name="in_clear">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>310</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>100</y>
       <width>391</width>
       <height>201</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>供货商</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>价格</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>数量</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>名称</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>条形码</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="in_code"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="in_name"/>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="in_qty"/>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="in_price"/>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="in_sup"/>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
